clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -226,6 +227,10 @@ public Password getPassword(String key) {
         return keys;
     }
 
+    public boolean originalsContainsKey(String configKey) {
```

Comment:
What's the functional difference between `config.originalsContainsKey("foo")` and `config.originals().containsKey("foo")`?

I understand the latter creates an unnecessary copy, but beyond that, does the logic differ?

Reply:
No logic difference. But anyway, I've reverted this change because AbstractConfig is a public class.